examples/tf-models-segmenter-i2v/README.md

```diff
@@ -0,0 +1,239 @@
+# Deploying an Instance Segmentation Model via the Image-To-Video Tool
+
+This directory demonstrates how to deploy an instance segmentation model from
+the [TensorFlow Detection Model Zoo](
+https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md)
+to the Voxel51 Platform via the Image-To-Video tool.
+
+
+## Overview
+
+This directory defines a `tf-models-segmenter-i2v` analytic that is ready for
+deployment to the Voxel51 Platform.
+
+The following files constitute the definition of the analytic:
+
+- `main.bash`: the main Docker entrypoint script, which calls `main.py` and
+gracefully handles any uncaught errors that are raised therein
+
+- `main.py`: the main executable for the analytic, which loads the model using
+the `eta.detectors.TFModelsSegmenter` interface, performs inference on each
+frame provided by the pre-processing container, and saves the resulting
+predictions as an `eta.core.video.VideoLabels` JSON file on disk
+
+- `analytic.json`: the analytic JSON file for the analytic, which declares
+the inputs and outputs that the analytic exposes
+
+- `Dockerfile`: the Dockerfile that specifies how to build an image containing
+the analytic and its dependencies
+
+
+## Environment setup
+
+First, install the Platform SDK with a full ETA installation:
+
+> Note: you probably want to do this in a fresh virtual environment so you
+> don't interfere with your existing ETA installation!
+
+```shell
+# Install Platform SDK
+git clone https://github.com/voxel51/platform-sdk
+cd platform-sdk
+git submodule init
+git submodule update
+
+# Full ETA install
+cd eta
+bash install.bash
+cp config-example.json config.json
+
+cd ../..
+```
+
+
+## Model setup
+
+### TF Detection Model Zoo
+
+Choose a model of interest from the [TF Detection Model Zoo](
+https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md):
+
+```shell
+#
+# Choose any model that outputs masks from
+# https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md
+#
+MODEL=mask_rcnn_resnet101_atrous_coco_2018_01_28
+```
+
+Download the model of interest and extract the frozen inference graph:
+
+```shell
+# Extracts frozen inference graph to `models/frozen_inference_graph.pb`
+wget -P models/ http://download.tensorflow.org/models/object_detection/${MODEL}.tar.gz
+tar -xf models/${MODEL}.tar.gz -C models/
+mv models/${MODEL}/frozen_inference_graph.pb models/frozen_inference_graph.pb
+rm models/${MODEL}.tar.gz
+rm -r models/${MODEL}
+```
+
+### ETA Models Registry
+
+Alternatively, you can directly download a frozen inference graph from the ETA
+Models Registry:
```

Comment:
Might help to have a link here

Reply:
The folder itself isn't publicly viewable, only the individual files, whose IDs are available via the models manifest JSON in ETA